docker/build_scripts/build_utils.sh

```diff
@@ -200,7 +200,8 @@ function build_curl {
 
 
 function do_standard_install {
-    ./configure > /dev/null
+    local configure_args=$1
+    ./configure "$configure_args" > /dev/null
```

Comment:
If I'm remembering correctly how shell functions work, I think this will only pass the _first_ argument to `do_standard_install` down to `./configure`.  The caller that needs this passes two arguments. I think what you actually want here is

    function do_standard_install {
        ./configure "$@" > /dev/null
        ...
    }

Reply:
Oops.. I only tested it with one argument and added the other args later so haven't noticed the issue.

Reply:
Also, I might be wrong, but I think that `"$configure_args"` will pass all the incoming arguments as a single argument containing spaces. I think you need it without the `""`.

(I'm by no means an expert on shell scripting, so I could be totally wrong here)

Reply:
@takluyver You're correct for `$configure_args`, but `$@` is special and pretty much always should be used as literally `"$@"`.

Reply:
TIL; thanks :-)